src/components/Pagination/Pagination.tsx

```diff
@@ -1 +1,61 @@
-export const Pagination = () => {};
+import { PaginationItem } from '../PaginationItem';
+
+interface Props {
+  total: number;
+  perPage: number;
+  currentPage: number;
+  setCurrentPage: (page: number) => void;
+}
+
+export const Pagination: React.FC<Props> = ({
+  total,
+  perPage,
+  currentPage,
+  setCurrentPage,
+}) => {
+  const itemsPerPage = Math.ceil(total / perPage);
+
+  const pages = Array.from({ length: itemsPerPage }, (_, i) => i + 1);
+
+  return (
+    <ul className="pagination">
+      <li className={`page-item ${currentPage === 1 ? 'disabled' : ''}`}>
+        <a
+          data-cy="prevLink"
+          className="page-link"
+          href="#prev"
+          aria-disabled={`${currentPage === 1}`}
+          onClick={() => currentPage !== 1 && setCurrentPage(currentPage - 1)}
+        >
+          «
+        </a>
+      </li>
+
+      {pages.map((page, index) => {
+        return (
+          <PaginationItem
+            page={page}
+            currentPage={currentPage}
+            setCurrentPage={setCurrentPage}
+            key={index}
```

Comment:
Using the index as a `key` is not recommended if the list can change because it does not uniquely identify the item. If the `page` value is unique, consider using it as the `key` instead.